packages/vm/src/evm/eei.ts

```diff
@@ -589,6 +589,10 @@ export default class EEI {
       return new BN(0)
     }
 
+    // EIP-2681 check
+    if (this._env.contract.nonce.eq(MAX_UINT64)) {
```

Comment:
would it be safer to use `gte`? or does it rather not matter since this would stop it from ever exceeding that value

Reply:
Yes, you are absolutely right!

Reply:
```suggestion
    if (this._env.contract.nonce.gte(MAX_UINT64)) {
```